src/source/reference/put-campaign-linkedads.rst

```diff
@@ -0,0 +1,62 @@
+.. index:: PUT /campaign/{id}/linkedAds
+.. _put_campaign_linkedads:
+
+PUT /campaign/{id}/linkedAds
+============================
+
+It is possible to add/override a :ref:`single <single_linkedad>` linkedAd or :ref:`multiple <list_linkedad>` linkedAds to a campaign in one go.
+If the campaign already has a linkedAd with the same adId, the existing linkedAd will be overwritten with
+the provided one.
+It is not possible to have multiple linkedAds in one campaign with the same adId.
+Setting linkedAds on a campaign in a ``DELETED`` state is not allowed.
+
+A single linkedAd has the following request model structure:
+
+.. include:: ../examples/campaign-ad-request.rst
+
+.. _single_linkedad:
+
+Adding/overriding a single linkedAd:
+-------------------------------------
+
+.. list-table::
+ :widths: 20 80
+
+ * - Scope
+   - ``api_rw`` or ``console_rw``
+
+ * - Accept
+   - ``application/sellside.campaign.linkedad-v1+json; charset=utf-8``
+
+ * - Content-Type
+   - ``application/sellside.campaign.linkedad-v1+json; charset=utf-8``
+
+Adds a linkedAd to the campaign or overrides ones with same adIds. If successful, the server responds with `HTTP 200`.
```

Comment:
overriding -> modifying

Reply:
I like `replacing` better, it indicates that it's a ful overwrite.